tests/unit/core/components/TestInstallButton.js

```diff
@@ -34,11 +34,13 @@ describe('<InstallButton />', () => {
     getClientCompatibility: () => ({ compatible: true }),
     hasAddonManager: true,
     i18n: getFakeI18nInst(),
+    store: createStore().store,
     ...customProps,
   });
 
-  const render = (props) =>
-    shallow(<InstallButtonBase {...renderProps(props)} />);
+  const render = (props) => shallowUntilTarget(
+    <InstallButton {...renderProps(props)} />, InstallButtonBase
```

Comment:
Would it be possible to know how many HOCs we're `compose`-ing and just get that element or do we need to have the base component as well?